docs/user/debugging.md

```diff
@@ -34,3 +34,49 @@ adb shell am start -n org.mozilla.samples.glean/mozilla.components.service.glean
 - There are a couple different ways in which to send pings through the GleanDebugActivity.
     1. You can use the `GleanDebugActivity` in order to tag pings and trigger them manually using the UI.  This should always produce a ping with all required fields.
     2. You can use the `GleanDebugActivity` to tag _and_ send pings.  This has the side effect of potentially sending a ping which does not include all fields because `sendPings` triggers pings to be sent before certain application behaviors can occur which would record that information.  For example, `duration` is not calculated or included in a baseline ping sent with `sendPing` because it forces the ping to be sent before the `duration` metric has been recorded.
+
+## Glean Log messages
+
+Glean logs warnings and errors through the Android logging framework.
+When running a Glean-powered app in the Android emulator or on a device connected to your computer via cable, there are several ways to read the log output.
+
+### Android Studio
+
+Android Studio can show the logs of a connected emulator or device.
+To display the log messages for an app:
+
+1. Run an app on your device.
+2. Click **View > Tool Windows > Logcat** (or click **Logcat** in the tool window bar).
+
+The Logcat window will show all log messages and allows to filter those by the application ID.
+Select the application ID of the product you're debugging.
+
+More information can be found in the [View Logs with Logcat][] help article.
+
+[View Logs with Logcat]: https://developer.android.com/studio/debug/am-logcat
```

Comment:
It is worth mentioning that you can also filter on "Glean"?

Reply:
Good point. Let me add that.